src/main/java/com/cburch/logisim/std/arith/ArithmeticLibrary.java

```diff
@@ -54,6 +54,14 @@
     new FactoryDescription(Shifter.class, S.getter("shifterComponent"), "shifter.gif"),
     new FactoryDescription(BitAdder.class, S.getter("bitAdderComponent"), "bitadder.gif"),
     new FactoryDescription(BitFinder.class, S.getter("bitFinderComponent"), "bitfindr.gif"),
+    new FactoryDescription(FPAdder.class, S.getter("fpAdderComponent"), "adder.gif"),
```

Comment:
I am not sure about the `fp` prefix here. It looks clear now in context of this PR but otherwise might be bit cryptic. From the other hand i.e. `float` might be misleading too as this not about float as floating point. Not sure. @BFH-ktt1 @maehne your thoughts?